.github/workflows/release.yml

```diff
@@ -144,22 +123,301 @@ jobs:
         .\scripts\ValidatePackageContents.ps1 -ArtifactId microsoft-kiota-serialization-text -Version $env:PACKAGE_VERSION
         .\scripts\ValidatePackageContents.ps1 -ArtifactId microsoft-kiota-serialization-multipart -Version $env:PACKAGE_VERSION
         .\scripts\ValidatePackageContents.ps1 -ArtifactId microsoft-kiota-bundle -Version $env:PACKAGE_VERSION
-    - name: Publish Release abstractions #publishing all components at once often results in split staging repos which fails to release
+
+  release-abstractions-maven-central:
```

Comment:
please use a matrix instead, the steps are identical to the component name and the relative path for the copy of secring